### ISSUE TITLE
Custom launcher flags in wrong position

### ISSUE THREAD

**Issue by MattAtBT** [karma-runner/karma-phantomjs-launcher#45]
In my karma.conf.js, I am trying to pass a time zone flag to the launcher.

```
  customLaunchers: {
      'PhantomJS_custom': {
        base: 'PhantomJS',
        flags: ['TZ=UTC']
      }
    }
```

This generates a command like 
`
/path/to/phantomjs TZ=UTC /path/to/test-script.js
`

But that command is invalid. The valid order of flags is:
`
/path/to/phantomjs /path/to/test-script.js TZ=UTC 
`

**Comment by gskachkov** [comment]
@MattAtBT  I've added fix to this issue. It will be available as soon as it is merged, also you can use **nmp link** to switch branch with my fix

**Comment by Dignifiedquire** [comment]
Fixed in #91
